### PR TITLE
Task4

### PR DESCRIPTION
**Pull request by andreyv24**

### CODE REVIEW

File: src/main/java/com/walking/intensive/chapter1/task4/Task4.java [hunk 1]
```diff
@@ -23,19 +23,31 @@
...
-        double c = 0;
+        double a = 5;
+        double b = -4;
+        double c = -1;
 
         System.out.println(solveEquation(a, b, c));
-
     }
-
     static String solveEquation(double a, double b, double c) {
```
Comment (Zawadskiy) [inline src/main/java/com/walking/intensive/chapter1/task4/Task4.java:35]: Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter1/task1/Task1.java:23:36: ')' is preceded with whitespace. [ParenPad]
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter1/task4/Task4.java:1: File does not end with a newline. [NewlineAtEndOfFile]
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter1/task4/Task4.java:39:44: '/' is not preceded with whitespace. [WhitespaceAround]
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter1/task4/Task4.java:47:13: '}' at column 13 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally). [RightCurly]
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter1/task4/Task4.java:48:13: 'else' construct must use '{}'s. [NeedBraces]

Чекстайл исправить. И почему здесь две таски в pr?

Comment (Zawadskiy) [reply src/main/java/com/walking/intensive/chapter1/task4/Task4.java:35]: Ну и еще тестіі пройди =)
> Task :test FAILED

Task1Test > getAgeStringTest(String, Integer) > [4] Вам 0 лет, 0 FAILED
    org.opentest4j.AssertionFailedError at Task1Test.java:19

Task1Test > getAgeStringTest(String, Integer) > [131] Вам 127 лет, 127 FAILED
    org.opentest4j.AssertionFailedError at Task1Test.java:19

Task1Test > getAgeStringTest(String, Integer) > [1[32](https://github.com/KFalcon2022/intensive-tasks-2024/actions/runs/11923251205/job/33232845433#step:6:33)] Вам 19875 лет, 19875 FAILED

    org.opentest4j.AssertionFailedError at Task1Test.java:19

Task1Test > getAgeStringTest(String, Integer) > [1[33](https://github.com/KFalcon2022/intensive-tasks-2024/actions/runs/11923251205/job/33232845433#step:6:34)] Вам 6951 год, 6951 FAILED
    org.opentest4j.AssertionFailedError at Task1Test.java:19

Task1Test > getAgeStringTest(String, Integer) > [1[34](https://github.com/KFalcon2022/intensive-tasks-2024/actions/runs/11923251205/job/33232845433#step:6:35)] Вам 356782 года, 356782 FAILED
    org.opentest4j.AssertionFailedError at Task1Test.java:19

Task4Test > solveEquationTest(String, double, double, double) > [1] Количество решений: 0., 0, 0, -10 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [2] Количество решений: 0., 0, 0, 1 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [3] Количество решений: 0., 0, 0, 4 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [4] Бесконечное множество решений., 0, 0, 0 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [5] Количество решений: 0., 1, 0, 1 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [6] Количество решений: 0., 2, 0, 2 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [7] Количество решений: 0., 3, 0, 3 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [8] Количество решений: 0., 4, 2, 5 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [9] Количество решений: 0., 5, 1, 6 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [10] Количество решений: 0., 6, -1, 8 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [11] Количество решений: 0., 7, 3, 10 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [12] Количество решений: 0., 8, 4, 20 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [13] Количество решений: 0., 9, 5, 30 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [14] Количество решений: 0., 10, 6, 40 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [15] Количество решений: 1. Корень: 1.0, 0, -2, 2 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [16] Количество решений: 1. Корень: 2.0, 0, -2, 4 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [17] Количество решений: 1. Корень: 4.0, 0, -2, 8 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [18] Количество решений: 1. Корень: 2.0, 0, 20, -40 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [19] Количество решений: 1. Корень: 8.0, 0, 10, -80 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [20] Количество решений: 1. Корень: 0.0, 1, 0, 0 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [21] Количество решений: 1. Корень: 1.0, 1, -2, 1 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [22] Количество решений: 1. Корень: 2.0, 1, -4, 4 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [23] Количество решений: 1. Корень: 3.0, 1, -6, 9 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [24] Количество решений: 1. Корень: 4.0, 1, -8, 16 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [25] Количество решений: 1. Корень: 5.0, 1, -10, 25 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [26] Количество решений: 2. Корни: -5.0;-4.0, 1, 9, 20 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [27] Количество решений: 2. Корни: -3.0;-2.0, 1, 5, 6 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [28] Количество решений: 2. Корни: 3.0;5.0, 1, -8, 15 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [29] Количество решений: 2. Корни: -9.0;5.0, 1, 4, -45 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18

Task4Test > solveEquationTest(String, double, double, double) > [30] Количество решений: 2. Корни: -1.0;4.0, -1, 3, 4 FAILED
    org.opentest4j.AssertionFailedError at Task4Test.java:18
164 tests completed, [35](https://github.com/KFalcon2022/intensive-tasks-2024/actions/runs/11923251205/job/33232845433#step:6:36) failed

Comment (Zawadskiy) [reply src/main/java/com/walking/intensive/chapter1/task4/Task4.java:35]: Подсказка: посмотри какой віівод должен бііть.

**Review by Zawadskiy — CHANGES_REQUESTED** [review]

**Comment by Zawadskiy** [comment]
Срок внесения правок вышел, при необходимости ревью - откройте новый Pull Request

**Comment by andreyv24** [comment]
ОК. Внёс правки, открыл новый PR.

**Pull request closed without merging**